lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
+const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   create() {
     this.serverless.cli.log('Creating Stack...');
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+    let stackTags = { STAGE: this.options.stage };
```

Comment:
Ok that makes sense, didn't know this use case. With that in mind it makes sense to keep this in